lib/api-blueprint/cache.rb

```diff
@@ -16,10 +16,10 @@ def write(id, data, options)
       data
     end
 
-    def generate_cache_key(options)
+    def generate_cache_key(klass, options)
       options = options.clone.except :body
       options_digest = Digest::MD5.hexdigest Marshal::dump(options.to_s.chars.sort.join)
-      "#{key}:#{options_digest}"
+      "#{key}:#{klass&.name}:#{options_digest}"
```

Comment:
this could possibly end up `{key}::{hash}` if `klass` is missing, but that may be ok? Or should we default it to something `{key}:ApiBlueprintUnknown:{hash}`?

Reply:
It could, yeah, but I don't think it's an issue. It's only really possible if you're using the runner without a model, but I think that's really an edge case. In that scenario, having a key with a double "::" doesn't really seem like an issue. It didn't feel worth the code require to work around it.. Or?

Reply:
No, I agree... commented  more to just check it had been considered 👍

merge away! :shipit:

Reply:
👍